src/index.ts

```diff
@@ -241,7 +241,7 @@ const txBodies = async (req: Request, res: Response) => {
 const getStatus = async (req: Request, res:  Response) => {
   const mobilePlatformVersionPrefixes = ["android / ", "ios / ", "- /"];
   const clientVersionHeader = 'yoroi-version';
-  const minMobileVersion = '2.2.2';
+  const minMobileVersion = '2.2.3';
```

Comment:
@mebassett I think this should be 2.2.4, so 2.2.3 is still shown in maintenance

Reply:
yeah either that or we can check the sermver check to be <= instead of <